daemon-tests/tests/happy_path.rs

```diff
@@ -412,20 +419,24 @@ async fn double_rollover_an_open_cfd() {
     rollover_an_open_cfd(Position::Short, 2).await;
 }
 
-async fn rollover_an_open_cfd(maker_position: Position, nr_rollovers: u8) {
+async fn rollover_an_open_cfd(maker_position: Position, nr_rollovers: i64) {
```

Comment:
nit: IMHO `u8` chosen by you earlier was better due type safety guarantees, e.g.that no-one could try out e. `-1` value. Perhaps consider casting to `i64` internally? (it cannot fail from such a small number)

Reply:
I only changed it because I needed a an `i64` further down in the code. I don't think it matter much but I'm happy to cast instead.

Reply:
I figured :) it's probably cleaner to cast it when you use it instead of changing the API (not a biggie though, it's just a test...).